addons/event_sale/security/ir.model.access.csv

```diff
@@ -3,3 +3,6 @@ access_event_event_ticket_user,event.event.ticket.user,event_sale.model_event_ev
 access_event_event_ticket_admin,event.event.ticket.admin,event_sale.model_event_event_ticket,event.group_event_manager,1,1,1,1
 access_product_template_event_manager,product.template.event.manager,product.model_product_template,event.group_event_manager,1,1,1,1
 access_product_product_event_manager,product.product.event.manager,product.model_product_product,event.group_event_manager,1,1,1,1
+"access_registration_editor","access.registration.editor","model_registration_editor","event.group_event_user",1,0,1,0
+"access_registration_editor_line","access.registration.editor.line","model_registration_editor_line","event.group_event_user",1,1,1,1
+"access_event_event_configurator","access.event.event.configurator","model_event_event_configurator","event.group_event_user",1,0,1,0
```

Comment:
these should probably be for `sales_team.group_sale_salesman` instead, it's triggered when editing a SO (`event.event.configurator`) or confirming a quote (`registration.editor(.line)`)

Reply:
they are used to fill information regarding the ticket being sold, then regarding the attendees upon confirmation. They happen only when dealing with the Sales app and should not require access to Event (note that Event User is an implied group of 'Sales: See Own Leads', so some access is still automatically there to read events, etc. so just switching the group of these ACLs should work OoB)